src/Aspire.Dashboard/Components/Pages/Resources.razor.cs

```diff
@@ -154,7 +155,7 @@ protected override async Task OnInitializedAsync()
             new GridColumn(Name: NameColumn, DesktopWidth: "1.5fr", MobileWidth: "1.5fr"),
             new GridColumn(Name: StateColumn, DesktopWidth: "1.25fr", MobileWidth: "1.25fr"),
             new GridColumn(Name: StartTimeColumn, DesktopWidth: "1fr"),
-            new GridColumn(Name: TypeColumn, DesktopWidth: "1fr"),
+            new GridColumn(Name: TypeColumn, DesktopWidth: "1fr", IsVisible: () => _showResourceTypeColumn),
```

Comment:
What do folks feel about having an option to still show the type column? Would anyone use it?

Reply:
The main reason I didn't just delete it was that I didn't have a better default sort column unless we made the Icon its own column. It's less usable as it is now in terms of getting back to default sort order if you sort by something else, you would need to make the column visible and then sort by it to get back to the default.

I don't feel strongly about this though. Just seemed like not super useful when the information was there as tooltips and icons.

Reply:
If we hide and show this column then it should be persisted to local storage. That means visibility is preseved when starting/stopping Aspire.